packages/core/directives/control.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, ElementRef, Input, OnChanges, SimpleChanges, ViewEncapsulation} from '@angular/core';
+import {GoogleMapsAPIWrapper} from '../services/google-maps-api-wrapper';
+import {ControlPosition} from '../services/google-maps-types';
+
+/**
+ * AgmMapControl allows to add a custom control to the map
+ *
+ * ### Example
+ *
+ * ```
+ * <agm-control [position]="position" #my-id>
```

Comment:
please remove `#my-id` and use `[position]="'TOP_RIGHT'"`

Reply:
@SebastianM that example won't work because because google maps library the integer value of the position not the literal.